airflow/gcp/hooks/gcs.py

```diff
@@ -323,6 +323,26 @@ def delete(self, bucket_name, object_name):
 
         self.log.info('Blob %s deleted.', object_name)
 
+    def delete_bucket(self, bucket_name: str):
+        """
+        Delete an bucket object from the Google Cloud Storage
```

Comment:
Does it work on non empty buckets?

Reply:
After our discussion about it, now it works.